Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.403
+FROM mcr.microsoft.com/dotnet/sdk:5.0
 
 # Dockerfile meta-information
 LABEL maintainer="NOS Inovação S.A." \
-    app_name="dotnet-sonar"
+    app_name="dotnet-sonar-50"
 
-ENV SONAR_SCANNER_MSBUILD_VERSION=4.10.0.19059 \
+ENV SONAR_SCANNER_MSBUILD_VERSION=5.0.3.23901 \
     DOTNETCORE_SDK=3.1.403 \
-    DOTNETCORE_RUNTIME=3.1.9 \
-    NETCOREAPP_VERSION=netcoreapp3.0 \
+    DOTNETCORE_RUNTIME=5.0 \
```

Comment:
This should always contain the exact information of the contents of the DockerImage:
```
    DOTNETCORE_SDK=5.0.100 \
    DOTNETCORE_RUNTIME=5.0.0 \
    NETAPP_VERSION=net5.0 \
```
ref.: https://dotnet.microsoft.com/download/dotnet/5.0

Reply:
Done